.github/workflows/stale_pull_requests.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Close stale pull requests'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

Comment:
nit: do we need to run it so frequently :-P

Reply:
actually nvm. doesn't hurt since it is so lightweight

Reply:
Isn't this just daily? That seems like the minimum viable frequency, since this will probably start closing multiple PRs per day after it gets through the backlog.

Actually at the default operations-per-run value of 30, we'll need to run this workflow dozens of times in a row to get through the backlog.

Reply:
yeah I was thinking since we are doing a 30-60, might as well just do weekly or sth during the weekend.